README.md

```diff
@@ -130,12 +132,14 @@ parameters.
 Configuration
 -------------
 
-* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_FAILURE` (default `1`)
-* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_SUCCESS` (default `3`)
-* `BACKUP_K8S_CRONJOB_BACKUP_SCHEDULE` (default `"0 0 * * *"`, once a day at 
-  midnight)
-* `BACKUP_K8S_CRONJOB_RESTORE_SCHEDULE` (default `None`, periodic restore is 
-  disabled)
+* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_FAILURE` (default: `1`)
+* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_SUCCESS` (default: `3`)
+* `BACKUP_K8S_CRONJOB_STARTING_DEADLINE_SECONDS` (default: `60`)
```

Comment:
60 seconds feels a little tight for a cron job that conceivably runs a few times a day, if even that. We've had backup jobs evicted from their node on account of disk pressure and suceed to start on their 3rd or 4th try, after about 10-15 minutes. If i understand correctly how CronJob deadlines work, then setting it to 60 would just cause the scheduler to give up if it can't launch the job within 60 seconds. Maybe we set this to something like 900 seconds (15 minutes) by default?

Reply:
Yes that sounds reasonable. I'll change that to 900.